static/docs/user-guide/install/index.md

```diff
@@ -0,0 +1,62 @@
+# Install
+
+There are a few ways to install DVC: using `pip` or `conda`, which are system
+agnostic, and using packages/installers for specific systems.
+
+> It may happen that you install DVC first for testing and then try to install
+> it again via a different way. This may cause any conflicts or confusion. So,
+> before trying to install it, make sure that it is not already installed, using
+> a command like: `which dvc`.
```

Comment:
Good note! But let's make it more compact? Something like:

Please double check you don't already have DVC before trying to install it (again), with a command like `which dvc`.